frontend/src/components/Calendar/UserSearch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import styles from './UserSearch.module.css';
+
+const UserSearch = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const handleSearchChange = (e) => {
+    setSearchQuery(e.target.value);
+  };
+
+  const handleSearchSubmit = (e) => {
+    e.preventDefault();
+    // Perform the search action using the searchQuery value
+    // TO-DO: implement your search logic here
+    console.log(`Search query: ${searchQuery}`);
+  };
+
+  return (
+    <div>
+        <h2 className={styles.button1}>Meet with...</h2>
```

Comment:
- Kinda same question with the Today's Task button. I assume this button will bring us to the study with me page?

Reply:
Yes or invite other users to collaborate on a task, but link to study room is good